ui/src/lib/ImageFinder.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
```

Comment:
this ImageFinder.svelte is completely different from this what is on main.
On main is new version of file in lib/components